components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -346,9 +347,22 @@ class GeckoEngine(
             } ?: emptyList()
 
             extensions.forEach { extension ->
+                // https://github.com/mozilla-mobile/android-components/issues/6227 set all installed
```

Comment:
Let's link to this one instead, I think that's clearer: As a workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=1621385 we set all installed extensions....

Reply:
Unresolved this as the change didn't get applied.